From e81637e18e458798ecf43e7770fec41d938bef3c Mon Sep 17 00:00:00 2001
From: Hartmut Knaack <knaack.h@gmx.de>
Date: Thu, 17 Nov 2016 22:49:10 +0100
Subject: [PATCH] dts: change lamobo r1 phy-mode

Change phy-mode of the b53125 present on Lamobo R1 to rgmii-txid.
---
 arch/arm/boot/dts/sun7i-a20-lamobo-r1.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/sun7i-a20-lamobo-r1.dts b/arch/arm/boot/dts/sun7i-a20-lamobo-r1.dts
index 73c05da..4a7692a 100644
--- a/arch/arm/boot/dts/sun7i-a20-lamobo-r1.dts
+++ b/arch/arm/boot/dts/sun7i-a20-lamobo-r1.dts
@@ -167,7 +167,7 @@
 					reg = <8>;
 					label = "cpu";
 					ethernet = <&gmac>;
-					phy-mode = "rgmii";
+					phy-mode = "rgmii-tx";
 					fixed-link {
 						speed = <1000>;
 						full-duplex;
-- 
2.7.4

